frontend/src/laws/InitialAtomausstieg.ts

```diff
@@ -0,0 +1,22 @@
+import { createLaw } from "../Factory"
+
+export default createLaw({
+  title: "Atomausstieg finded wie beschlossen 2022 statt",
+  description: "",
+  labels: ["hidden", "initial"],
+
+  effects(data, startYear, currentYear) {
+    const newValue = () => {
+      if (currentYear == 2021) {
+        return 60.45
+      }
+      if (currentYear == 2022) {
+        return 30.21
+      }
+      return 0
+    }
```

Comment:
Verstehe noch nicht ganz, was es mit der Funktion auf sich hat. Eigentlich wird doch nur ein Wert gebraucht, der abhängig ist vom Jahr, oder? Dann wäre es so was wie

```
const mapping = {
  2021: 60.45,
  2022: 30.21,
}
const newValue = mapping[currentYear] || 0
```
Bei Vergleichen sind übrigens die  drei Gleichheitszeichen (`===` ) zu empfehlen, da die typsicher sind
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons#answer-359509

Reply:
Ich lerne noch. Danke!

Reply:
Da bekomme ich  folgenden Fehler angezeigt:
Element implicitly has an 'any' type because expression of type 'number' can't be used to index type '{ 2021: number; 2022: number; }'.
  No index signature with a parameter of type 'number' was found on type '{ 2021: number; 2022: number; }'.ts(7053)

Reply:
Danke! Habs verbessert, dabei den Typ oben mit Doppelpunkt angeben...